packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -217,10 +238,11 @@ export abstract class TelemetryLogger implements ITelemetryLoggerExt {
 		return this.extendProperties(newEvent, includeErrorProps);
 	}
 
-	private extendProperties<T extends ITelemetryPropertiesExt>(
-		eventLike: T,
+	private extendProperties<T extends ITelemetryLoggerPropertyBag = ITelemetryLoggerPropertyBag>(
+		toExtend: T,
```

Comment:
i didn't really change this code, i just moved it, it actually wasn't even necessary to move, it was an artifact of some other changes i was porting over from a different branch. if you look at prepare event, which is the only place this is called, and where it used to live, that code takes ITelemetryBaseEvent which extents ITelemetryLoggerPropertyBag, so i think this change is correct.

 I agree the typing is messed up here. It actually looks like a problem with ITelemetryPropertiesExt which probably should support getters, @markfields who should probably look into that.

Reply:
i don't actually get ITelemetryPropertiesExt, or how that works with general object vs tagged objects, seems complicated, i'm also not sure i see the benefit to logging arrays and objects.

Reply:
Yeah the property bag types don't have Ext verions.  See https://dev.azure.com/fluidframework/internal/_queries/edit/4552, it's a leftover/low-pri todo from the work to add array support.

Tony, happy to discuss the "why" of supporting arrays and objects elsewhere

Reply:
@anthony-murphy why isn't it just this?  There's only one caller and it passes an `ITelemetryBaseEvent`.

```suggestion
	private extendProperties(eventLike: ITelemetryBaseEvent,
```

Reply:
its because i shouldn't have originally include it. i want to add a function to pull the properties off a logger to add to errors, which will reuse this, and has really been my end goal all along :)